luaui/RmlWidgets/gui_awards/gui_awards.rcss

```diff
@@ -0,0 +1,171 @@
+* {
+    font-family: "Exo 2";
+    font-size: 18px;
+    color: white;
+}
+
+body {
+  width: 100%;
+  height: 100%;
+  pointer-events: none;
+  background-color: rgba(255,255,255,0.7)
+}
+
+table {
+  box-sizing: border-box;
+  display: table;
+}
+
+tr {
+  box-sizing: border-box;
+  display: table-row;
+}
+
+td {
+  box-sizing: border-box;
+  display: table-cell;
+  text-align: center;
+  vertical-align: top;
+}
+
+thead, tbody, tfoot {
+  display: table-row-group;
+}
+
+body, div, p, hr, pre,
+h1, h2, h3, h4, h5, h6,
+tabset tabs {
+    display: block;
+}
+
+h1, h2, h3 {
+  font-weight: bold;
+}
+
+img {
+  height: 100px;
+
+}
+
+span {
+  display: inline-block;
+  margin: 0;
+  padding: 0;
+}
+
+p, span {
+  vertical-align: center;
+}
+
+td.align-left {
+  text-align: left;
+  vertical-align: middle;
+}
+
+#main-container {
+  position: absolute; 
+  left: 50%; 
+  top: 50%; 
+  transform: translateX(-50%) translateY(-50%);
+  pointer-events: auto;
+  min-width: 750px; 
+  border-radius: 10px; 
```

Comment:
Do not use `px` values ever. Always use `dp`. `dp` scales with the gui ratio established in `rml_context_manager.lua` this way when the scale and resolution of the screen changes, so to you retain positions and widths etc relative to the screen.